modules/tensor_mechanics/include/transfers/VectorPostprocessorToAuxVar.h

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
```

Comment:
I'm looking at the other transfers to see if the naming convention for this matches those, and it seems that something like 'VectorPostprocessorTransfer' might be more consistent. None of the transfers have anything about what kind of thing they are transferring to in their name.